react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.stories.tsx

```diff
@@ -73,30 +73,35 @@ CustomDashedWells.args = {
         {
             "@@type": "WellsLayer",
             data: "@@#resources.wellsData",
-            dashed: [10, 3],
+            style: { color: [255, 0, 0, 255], dash: [10, 3] },
             refine: false,
             outline: false,
         },
     ],
 };
-CustomDashedWells.parameters = {
+CustomStyledWells.parameters = {
     docs: {
         description: {
-            story: "Volve wells example with dashed well trajectories, with custom style.",
+            story: "Volve wells example with dashed style and red trajectories, with custom style.",
         },
         inlineStories: false,
         iframeHeight: 500,
     },
 };
 
 function callback(object: Record<string, Record<string, unknown>>) {
-    if (object["properties"]["name"] == "15/9-F-11 B") return [5, 3];
-    return [0, 0];
+    if (object["properties"]["name"] == "15/9-F-11 B")
+        return { color: [255, 0, 0, 255], dash: true };
+    else if (object["properties"]["name"] == "15/9-F-14")
+        return { dash: [10, 1] };
+    else if (object["properties"]["name"] == "15/9-F-10")
+        return { color: [0, 0, 0, 0] };
```

Comment:
I think the use case was to hide well trajectories and not well heads. If we can instead supply separate `trajectoryColor` and `headColor` then we can support hiding well head and/or trajectory separately

Reply:
yes, currently wellhead and trajectory color pic up color from same color prop, if we can separate the colors, it should be possible.